process/unsigned/feeTxHandler.go

```diff
@@ -0,0 +1,236 @@
+package unsigned
+
+import (
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/data"
+	"github.com/ElrondNetwork/elrond-go/data/block"
+	"github.com/ElrondNetwork/elrond-go/data/feeTx"
+	"github.com/ElrondNetwork/elrond-go/hashing"
+	"github.com/ElrondNetwork/elrond-go/marshal"
+	"github.com/ElrondNetwork/elrond-go/process"
+	"math/big"
+	"sync"
+)
+
+const communityPercentage = 0.1 // 1 = 100%, 0 = 0%
```

Comment:
Would it be better to have all of this economics related constants (including `MinGasPrice`, `MinTxFee`) in one place so it would be easier to find and change when needed?

Reply:
ok. agreed.